apps/hubble-stats/constants/index.ts

```diff
@@ -1,9 +1,46 @@
 import vAnchorClient from '@webb-tools/vanchor-client';
+import { getDateFromEpoch, getEpochFromDate } from '../utils';
 
-export const vAnchorAddresses = ['0x4b88368Eb14D7d09f0ca737832cEBfb8F12e3f05'];
+export const VANCHOR_ADDRESSES = ['0x7aA556dD0AF8bed063444E14A6A9af46C9266973'];
 
-export const allSubgraphUrls = [
+export const LIVE_SUBGRAPH_URLS = [
   vAnchorClient.SubgraphUrl.vAnchorOrbitAthena,
   vAnchorClient.SubgraphUrl.vAnchorOrbitHermes,
   vAnchorClient.SubgraphUrl.vAnchorOrbitDemeter,
 ];
+
+export const LOCAL_SUBGRAPH_URLS = [
+  vAnchorClient.SubgraphUrl.vAnchorAthenaLocal,
+  vAnchorClient.SubgraphUrl.vAnchorHermesLocal,
+  vAnchorClient.SubgraphUrl.vAnchorDemeterLocal,
+];
+
+export const ACTIVE_SUBGRAPH_URLS = LIVE_SUBGRAPH_URLS;
+
+export const EPOCH_DAY_INTERVAL = 24 * 60 * 60;
+
+export const EPOCH_NOW = getEpochFromDate(new Date());
+export const EPOCH_7D = EPOCH_NOW - 6 * EPOCH_DAY_INTERVAL;
```

Comment:
This should be a function, not a variable. The reason being, a variable is initialized when the file is used. However, we aim to acquire the date when this variable is in use, hence necessitating the use of a function.